skbio/diversity/__init__.py

```diff
@@ -399,10 +399,10 @@
 
 from skbio.util import TestRunner
 
-from ._driver import (alpha_diversity, beta_diversity,
+from ._driver import (alpha_diversity, beta_diversity, partial_beta_diversity,
```

Comment:
list `partial_beta_diversity` under "Functions" section of docstring above, that way it will be linked in the built docs